### PR TITLE
V2.1.0 rc1

### PR DESCRIPTION
**Pull request by paalonso**
Gents,

Opening MR. Everything should be ready for the SaaS multi-oe, hubs configs and AI. 

Asking formal review for merging.

TIA.

Rgds.

Pablo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ This repository is the source of truth for the OCI Open LZ and includes three ty
...
 | <img src="commons/images/size_xl.svg" width="23" align="center">  |...[`multi-tenancy/`](blueprints/multi-tenancy/) | Uses One-OE and Multi-OE to onboard all your organizations or customers into **several tenancies**.
-**2** |**[`addons/`](addons/)** | **Complement** your landing zone with add-ons to run OCI with best practices. (e.g., [Network Hubs](/addons/oci-hub-models/readme.md))|
-| **3** |**[`workload-extensions/`](workload-extensions/)** | **Extend** your landing zone with pluggable workload extensions (e.g., [EBS](/workload-extensions/ebs/readme.md), [OCVS](/workload-extensions/oci-lz-ext-ocvs/README.md), [...](/workload-extensions/readme.md)).|
+**2** |**[`addons/`](addons/)** | **Complement** your blueprint to run OCI with **best practices**. (e.g., [**Network Hubs**](/addons/oci-hub-models/readme.md))|
+| **3** |**[`workload-extensions/`](workload-extensions/)** | **Extend** your landing zone with pluggable **workloads** (e.g., [**EBS**](/workload-extensions/ebs/readme.md), [**OCVS**](/workload-extensions/oci-lz-ext-ocvs/README.md), [**AI**](/workload-extensions/ai-services/) [...](/workload-extensions/readme.md)).|
```
Comment (hrvolapeter) [inline README.md:29]: should we move the path `/workload-extensions` to `/workloads` as well?

File: addons/oci-oag/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+&nbsp; 
+
+# License
+
+Copyright (c) 2024 Oracle and/or its affiliates.
+
+Licensed under the Universal Permissive License (UPL), Version 1.0.
+
+See [LICENSE](LICENSE) for more details.
```
Comment (hrvolapeter) [inline addons/oci-oag/README.md:65]: /LICENSE

File: blueprints/multi-oe/saas/design/OCI_Open_LZ_Multi-OE_SaaS_Blueprint.drawio
Comment (hrvolapeter) [inline blueprints/multi-oe/saas/design/OCI_Open_LZ_Multi-OE_SaaS_Blueprint.drawio:1]: This file is too large (6.4MB), can you move to content branch

File: workload-extensions/ai-services/diagrams/ai-extension.gif
Comment (hrvolapeter) [inline workload-extensions/ai-services/diagrams/ai-extension.gif:1]: Can you move this to content branch?

**Review by hrvolapeter — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jpalmeiro) [reply README.md:29]: in the next iteration we can consider this

**Pull request merged**